package.json

```diff
@@ -27,7 +27,7 @@
     "build": "babel lib --out-dir dist",
     "prepublish": "npm run build",
     "lint": "eslint . --ignore-path .gitignore",
-    "coverage": "nyc --require babel-core/register _mocha -- $(find test -name '*.test.js')",
+    "coverage": "nyc --require babel-core/register _mocha -- test/*.test.js",
```

Comment:
I don't think this will recurse down the `test` directory. Doesn't really matter for now, but if we ever add sub-directories with tests it should probably be something like `test/**/*.test.js`.